packages/integration-youtube/src/index.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  createIconComponentFromSimpleIconsSvgPath,
+  getForegroundColor,
+  IntegrationConfig,
+} from '@techboi/consent-manager'
+
+import YouTube from 'simple-icons/icons/youtube'
+
+export { default as YouTubeVideo } from './youtube-video'
+
+export function YouTubeIntegration(): IntegrationConfig {
```

Comment:
I still think in a React domain it feels weird to use pascal case on a non-class non-component function.

Reply:
down for non-react = camel case

so `wrapperComponent` could become `WrapperComponent`

https://github.com/techboi/consent-manager/blob/ed8b18b8d276e188ec854db24ca98238f5bd3d0c/example/integrations/tracker-red-box-ltd.tsx#L94-L96